proposals/lip-0046.md

```diff
@@ -281,22 +286,23 @@ After all the transactions of a block `b` are executed, the following logic is a
 
 where the `isSeedValidInput` is the internal function specified above.
 
-### Block Creation
+### Block Generation
```

Comment:
A comment to the section above: I think, we have new hooks for the genesis block processing as well (see https://docs.google.com/document/d/1NzJt9Ygog7eXmND0a2fv1-PsZ1gCuZhu5328LTIe18Q/edit#heading=h.iuk4yqjhv61w). So you need to create the functions `verifyAssets` and `afterTransactionsExecute`.

Reply:
After checking in our internal channel, I think there is no need to define the functions in a pythonic way, since there is no change on the checks/executions.